### PR TITLE
[Feature] mention balance if not no prompt

### PR DESCRIPTION
**Pull request by camfairchild**
This PR adds a balance check and display during a transfer (without `--no_prompt`)  
image  
This should improve UX as a user can know how much TAO they have before sending

### CODE REVIEW

File: bittensor/_cli/__init__.py [hunk 1]
```diff
@@ -742,15 +742,27 @@ def check_transfer_config( config: 'bittensor.Config'):
...
-                console.print(":cross_mark:[red] Invalid Tao amount[/red] [bold white]{}[/bold white]".format(amount))
-                sys.exit()
+            if not config.no_prompt:
+                amount = Prompt.ask("Enter TAO amount to transfer")
+                try:
+                    config.amount = float(amount)
+                except ValueError:
+                    console.print(":cross_mark:[red] Invalid TAO amount[/red] [bold white]{}[/bold white]".format(amount))
+                    sys.exit()
+            else:
```
Comment (Eugene-hu) [inline bittensor/_cli/__init__.py:763]: What is this `else` for? if config.no_prompt == True?

Comment (camfairchild) [reply bittensor/_cli/__init__.py:763]: It's nested. It would be for `config.get('amount') is None and config.no_prompt == True`

Comment (Eugene-hu) [reply bittensor/_cli/__init__.py:763]: I see, so it errors out if the user doesn't specify an amount and picks no_prompt?

Comment (camfairchild) [reply bittensor/_cli/__init__.py:763]: exactly

**Review by Eugene-hu — APPROVED** [review]
LGTM

**Pull request merged**